ledger/heavy/executor/initial_state_keeper.go

```diff
@@ -162,6 +172,10 @@ func (isk *InitialStateKeeper) Get(ctx context.Context, lightExecutor insolar.Re
 	indexes := make([]record.Index, 0)
 
 	logger.Debugf("[ InitialStateKeeper ] Getting drops for: %s in pulse: %s", lightExecutor.String(), pulse.String())
+
+	// Exclude sending two equal drops to single LME after split
+	excludeDrops := make(map[insolar.JetID]bool)
```

Comment:
`excludeDrops` is confusing. We are not excluding them, just deduplicating. Something like `existingDrops` might be a better option. 
Also, we are only using it as a set. A common convention to do that is to use a map of empty structs.